hca2scea-backend/test/golden/arguments.csv

```diff
@@ -1,5 +1,13 @@
 spreadsheet,HCA project uuid,E-HCAD accession,curator initials,technology,experiment type,factor values,public release date,hca last update date,study accession
-Test01.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,50,AD,10Xv2_3,baseline,individual,2009-02-27,2021-04-29,SRP257542
-Test02.xlsx,2086eb05-10b9-432b-b7f0-169ccc49d270,52,AD,10Xv2_3,differential,organ part,2019-08-01,2021-03-01,SRP201293
-Test03.xlsx,d3a4ceac-4d66-4984-9704-2570c0647a56,54,AD,10Xv3_3,differential,organ part,2020-09-02,2021-03-01,SRP279608
-Test04.xlsx,559bb888-7829-41f2-ace5-2c05c7eb81e9,55,AD,10Xv2_3,differential,organ,2020-04-01,2021-03-01,SRP179761
+#Test01.xlsx,c893cb57-5c9f-4f26-9312-21b85be84313,50,AD,10Xv2_3,baseline,individual,2009-02-27,2021-04-29,SRP257542
+#Test02.xlsx,2086eb05-10b9-432b-b7f0-169ccc49d270,52,AD,10Xv2_3,differential,organ part,2019-08-01,2021-03-01,SRP201293
+#Test03.xlsx,d3a4ceac-4d66-4984-9704-2570c0647a56,54,AD,10Xv3_3,differential,organ part,2020-09-02,2021-03-01,SRP279608
+#Test04.xlsx,559bb888-7829-41f2-ace5-2c05c7eb81e9,55,AD,10Xv2_3,differential,organ,2020-04-01,2021-03-01,SRP179761
+GSE155468_ontologies.xlsx,07073c12-8006-4710-a00b-23abdb814904,36,AD,10Xv3_3,differential,organ,2021-09-30,2020-11-25,SRP274475
+# Wei's datasets
+210420Combined_ES_review.xlsx,955dfc2c-a8c6-4d04-aa4d-907610545d11,44,WKT,experiment type,factor values,public release date,hca last update date,GSE121638
+# Enrique's datasets
+# cannot find idf/sdrf files in scea's gitlab
+#GSE135893_ontologies_2020_07_15_redone_CellS_ES_MS.xlsx,c1a9a93d-d9de-4e65-9619-a9cec1052eaa,15,ESV,10Xv2_5,experiment type,factor values,public release date,hca last update date,GSE135893
+# no spreadsheet found for this one

```

Comment:
Do you need the spreadsheet for the command?

Reply:
Yes @ESapenaVentura the scripts uses the spreadsheet as well as the rest of the args to generate the idf/sdrf files.
You could send it to me, put in the ticket, or upload to the branch to `test/golden/<accession>/<accession>.xlsx`